ship.config.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  mergeStrategy: { toSameBranch: ['master'] },
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
+      return false;
+    }
+    return true;
+  },
+  versionUpdated: ({ version, dir }) => {
+    fs.writeFileSync(
+      path.resolve(dir, 'src', 'version.ts'),
+      `export const VERSION = '${version}';\n`
+    );
+  },
+  pullRequestTeamReviewer: ['instantsearch-for-websites'],
+  buildCommand: ({ version }) =>
+    `NODE_ENV=production VERSION=${version} yarn build`,
```

Comment:
Unlike IS.js, Angular-IS doesn't seem to use neither `process.env.VERSION` nor `process.env.NODE_ENV` in its build script. Do you think we can omit this config and use the default which is `yarn build`?

Reply:
Removed.